src/main/java/com/redhat/devtools/intellij/lsp4ij/LanguageServerWrapper.java

```diff
@@ -366,7 +366,7 @@ public synchronized void start() throws LanguageServerException {
                         final Map<URI, Document> toReconnect = filesToReconnect;
                         initializeFuture.thenRunAsync(() -> {
                             if (this.initialProject != null) {
-                                watchProject(this.initialProject, true);
+                                //watchProject(this.initialProject, true);
```

Comment:
why?

Reply:
I remove this code since we don't support workspace folder.